python/cudf/cudf/core/multiindex.py

```diff
@@ -33,7 +33,7 @@ class MultiIndex(Index):
     names: optional sequence of objects
         Names for each of the index levels.
     copy : bool, default False
-        Copy the meta-data. Not yet supported
+        Copy the meta-data.
```

Comment:
I know this is the Pandas docstring, but this doesn't seem accurate. What is the meta-data for a MultiIndex? If `copy=True` for us it seems like we're copying a DataFrame that's used to represent the codes of the MultiIndex?

Reply:
In pandas as they are copying levels and codes if copy is True, I can explicitly mention that copy create copy of codes.

Reply:
Got it. Yea lets mention that it creates a copy of the codes and/or levels instead.